package protoFile.guild.ModifyCorpsAnnoun;
message ModifyCorpsAnnounRequest{
     required int32 id = 1;//角色id
     required int32 corpsId = 2;//军团ID
     required string announContent = 3;//公告内容
}
message ModifyCorpsAnnounResponse{
     required bool result = 1;
     optional string message = 2;
}